config/env/alaninedipeptide.yaml

```diff
@@ -3,7 +3,8 @@ defaults:
 
 _target_: gflownet.envs.alaninedipeptide.AlanineDipeptide
 
-path_to_dataset: ${user.data.alanine_dipeptide}
+path_to_dataset: './data/alanine_dipeptide_conformers_1.npy'
```

Comment:
Isn't it better to have the previous, user-dependent path? (`${user.data.alanine_dipeptide}`)